projects/schematics/src/shared/utils/workspace-utils_spec.ts

```diff
@@ -88,14 +88,13 @@ describe('Workspace utils', () => {
     it('should return data about project', async () => {
       const workspaceInfo = getWorkspace(appTree);
       expect(workspaceInfo.path).toEqual('/angular.json');
-      expect(workspaceInfo.workspace.defaultProject).toEqual(appOptions.name);
     });
   });
 
   describe('getAngularJsonFile', () => {
     it('should return workspace', async () => {
       const workspace = getAngularJsonFile(appTree);
-      expect(workspace.defaultProject).toEqual(appOptions.name);
+      expect(workspace).not.toBeUndefined();
```

Comment:
What the change with the worksoace about?

Reply:
`defaultProject` deprecated and automatically removed after update to 14